src/tof/chopper.py

```diff
@@ -38,15 +39,19 @@ def __init__(
         close: sc.Variable,
         distance: sc.Variable,
         phase: sc.Variable,
+        direction: Literal['clockwise', 'anticlockwise'] = 'clockwise',
         name: str = "",
     ):
+        if frequency < (0.0 * frequency.unit):
```

Comment:
```suggestion
        if frequency <= (0.0 * frequency.unit):
```
I suppose you would get some division by zero otherwise?